iroh-net/src/magicsock.rs

```diff
@@ -287,6 +287,11 @@ impl Inner {
         let bytes_total: usize = transmits.iter().map(|t| t.contents.len()).sum();
         inc_by!(MagicsockMetrics, send_data, bytes_total as _);
 
+        let mut n = 0;
+        if transmits.is_empty() {
```

Comment:
part of me is curious how often this happens and would be tempted to log this on debug or trace level.  maybe also together with the `self.is_closed()` result as a field.

Reply:
just thought it made sense in preventing undesirable errors, but it's not like I've seen it before

Reply:
logs added in c8fa0de